frontera/contrib/scrapy/middlewares/seeds/__init__.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
A lot of the usage of absolute_import is preemptive. So it might not be needed for a lot of the cases. If you think it will affect performance, the one's which aren't needed can be removed.